lib/node/runner/jsonrpc.go

```diff
@@ -0,0 +1,177 @@
+package runner
```

Comment:
@Geod24 Under `api` it will also be good. At this time, jsonrpc handles only storage, but jsonrpc can handles various resources like node runner, so `runner` package is good place, I think.